Sources/TensorFlow/Loss.swift

```diff
@@ -290,3 +290,32 @@ public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
     let negAbsLogits = max(logits, -logits) // Custom `abs` to compute gradients at `0`.
     return reduction(maxLogitsWithZero - logits * labels + log1p(exp(-negAbsLogits)))
 }
+
+/// Returns the Huber loss between predictions and expectations.
+///
+/// For each value `x` in the difference `expected - predicted`, the loss is:
+/// - `0.5 * x^2` if `abs(x) <= δ`.
+/// - `0.5 * δ^2 + δ * (|x| - δ)` otherwise.
+///
+/// - Source: [Wikipedia article](https://en.wikipedia.org/wiki/Huber_loss).
+///
+/// - Parameters:
+///   - predicted: Predicted outputs from a neural network.
+///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+///   - delta: A floating point scalar representing the point where the Huber loss function changes
+/// from quadratic to linear.
```

Comment:
Please delete duplicate line. (I don't think GitHub supports multi-line suggestions, so committing my suggestions directly may not work.)
```suggestion
```

Reply:
I've just commit the fix. I hope there's no more typos I missed :-)